drivers/can/can_mcp2515.c

```diff
@@ -518,10 +518,10 @@ static int mcp2515_send(const struct device *dev,
 	return 0;
 }
 
-static int mcp2515_attach_isr(const struct device *dev,
-			      can_rx_callback_t rx_cb,
-			      void *cb_arg,
-			      const struct zcan_filter *filter)
+static int mcp2515_add_rx_filter(const struct device *dev,
+				 can_rx_callback_t rx_cb,
+				 void *cb_arg,
+				 const struct zcan_filter *filter)
 {
 	struct mcp2515_data *dev_data = DEV_DATA(dev);
 	int filter_idx = 0;
```

Comment:
Not critical but `filter_idx` would be better renamed to `filter_id` for consistency with `mcp2515_remove_rx_filter()` `filter_id` parameter naming update.

Reply:
Good catch. Renamed.